python/pfs/drp/datamodel/drp.py

```diff
@@ -0,0 +1,25 @@
+import pfs.datamodel.drp
+from .pfsSpectra import PfsSpectra
+from .pfsSpectrum import PfsSpectrum, PfsSimpleSpectrum
+
+__all__ = ("PfsArm", "PfsMerged", "PfsReference", "PfsSingle", "PfsObject")
+
+
+class PfsArm(pfs.datamodel.drp.PfsArm, PfsSpectra):
```

Comment:
We have the class `pfs.datamodel.drp.PfsArm` from the datamodel repository, and the subclass `pfs.drp.datamodel.drp.PfsArm` in the drp_stella repository. This is confusing. Both module names contain 'datamodel' and the drp_stella module has 'drp' twice, with no hint as to which repo it is in. Could the module name be different? For example `pfs.drp.stelladm.dmodel.PfsArm` for the drp_stella `PfsArm` subclass?